pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation_test.go

```diff
@@ -871,31 +940,38 @@ func injectAllEnvs() []corev1.EnvVar {
 			Name:  "DD_DOTNET_TRACER_HOME",
 			Value: "/datadog-lib",
 		},
-		{
-			Name:  "CORECLR_ENABLE_PROFILING",
-			Value: "1",
-		},
-		{
-			Name:  "CORECLR_PROFILER",
-			Value: "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}",
-		},
-		{
-			Name:  "CORECLR_PROFILER_PATH",
-			Value: "/datadog-lib/Datadog.Trace.ClrProfiler.Native.so",
-		},
```

Comment:
It's not intuitive but I think we still need these CORECLR variables even for the tracer to work. It's "safe" to inject those since they don't interfere with other language runtimes.

Reply:
@Kyle-Verhoog is it really just the `LD_PRELOAD` that I should drop here?

Reply:
correct, LD_PRELOAD is the problem affecting other runtimes